frontend/src/pages/dashboard/user-history/index.tsx

```diff
@@ -75,7 +75,7 @@ const UserHistory: FC<DashboardCompProps> = ({ userUid }) => {
                 dataSource={requests}
                 renderItem={(item) => (
                   <List.Item>
-                    <Card title={item.task.payerName} style={{ width: "100%" }}>
+                    <Card title={ `Payer ID: ${item.task.payerUid}`} style={{ width: "100%" }}>
```

Comment:
Is this another placeholder?

Reply:
Yes as mentioned will change to user display name